RadarSDK/RadarUtils.m

```diff
@@ -45,7 +45,7 @@ + (NSNumber *)timeZoneOffset {
 }
 
 + (NSString *)sdkVersion {
-    return @"3.3.0";
+    return @"3.3.1-beta.1";
```

Comment:
Remember, folks, the `.xcodeproj` file just says `3.3.1`, because TestFlight (which uses the project file's version number) doesn't support version strings like `3.3.1-beta.1`, so I have to use the bare version number, and indicate the beta by setting the `BUILD_NUMBER`.